snap/quota/resources_builder.go

```diff
@@ -35,8 +35,8 @@ type ResourcesBuilder struct {
 	CPUPercentage    int
 	CPUPercentageSet bool
 
-	AllowedCPUs    []int
-	AllowedCPUsSet bool
+	CPUSet    []int
+	CPUSetSet bool
```

Comment:
Ah, the wonders of the English language! :-)

Reply:
I'll admit it I didn't feel *great* when I looked at this haha!